# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: third_party/rust/mapped_hyph/src/lib.rs
# Commit: a1ab26abf3eb
# Full Hash: a1ab26abf3eb645e202d69dffea86398a0ea4563
# Author: Jonathan Kew <jkew@mozilla.com>
# Date: 2020-09-30 21:45:29
# Regressor Bug: 1667192
# File Overlap Count: 6
# Description:
#   Bug 1667192 - patch 1.1 - Run `mach vendor rust` to update in-tree mapped_hyph source. r=heycam
#   
#   Differential Revision: https://phabricator.services.mozilla.com/D91409
# ==============================================================================

diff -r 66b86f8b6682 -r a1ab26abf3eb third_party/rust/mapped_hyph/src/lib.rs
--- a/third_party/rust/mapped_hyph/src/lib.rs	Wed Sep 30 12:51:17 2020 +0000
+++ b/third_party/rust/mapped_hyph/src/lib.rs	Wed Sep 30 12:51:54 2020 +0000
@@ -430,7 +430,7 @@
 impl Hyphenator<'_> {
     /// Return a Hyphenator that wraps the given buffer.
     /// This does *not* check that the given buffer is in fact a valid hyphenation table.
-    /// Use is_valid_hyphenator() to determine whether it is usable.
+    /// Use `is_valid_hyphenator()` to determine whether it is usable.
     /// (Calling hyphenation methods on a Hyphenator that wraps arbitrary,
     /// unvalidated data is not unsafe, but may panic.)
     pub fn new(buffer: &[u8]) -> Hyphenator {
@@ -621,7 +621,7 @@
 ///
 /// # Safety
 ///
-/// This is unsafe for the same reason Mmap::map() is unsafe:
+/// This is unsafe for the same reason `Mmap::map()` is unsafe:
 /// mapped_hyph does not guarantee safety if the mapped file is modified
 /// (e.g. by another process) while we're using it.
 ///
